src/platform/functions/index.js

```diff
@@ -12,11 +12,20 @@ exports.scrapeEventbrite = functions
     timeoutSeconds: 60,
     memory: '1GB',
   })
-  .https.onRequest(async (_, res) => {
+  .https.onRequest(async (req, res) => {
     functions.logger.info('Starting to scrape...');
     let eventsArray = [];
     let collectiveEventsArray = [];
 
+    let city = '';
+    if (req.method === 'GET') {
+      if (req.query.city.length !== 0 && typeof req.query.city !== 'undefined') {
+        city = req.query.city;
+        functions.logger.info('City: ' + city);
+      } else {
+        functions.logger.error('No city name provided');
+      }
+    }
     /** This is a bug with firestore. For some reason, if you try to write to a nested
      * collection, it will not work, unless you write to the parent collection first. */
     await admin.firestore().collection('events').doc('scraped-events').set({});
```

Comment:
```suggestion
```
while we are here, let's remove this line as well. I realized what would happen is that every time we re scrape it will remove all the events in scraped-events. Instead what we will do in the future is initially it manually in firestore if necessary.

Reply:
removed!